### PR TITLE
Humble version. Tested ok.

### PR DESCRIPTION
**Pull request by jclinton830**
Thanks for these handy repo man!

The ROS2 branch within this repo is not built. So here is a modified version that was tested in humble.

### CODE REVIEW

**Comment by tomlogan501** [comment]
Oh nice !!! I asked on the issues if somebody want it but no one respond.
EDIT : not related, it is on the pointcloud_merger

File: README.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 # Robosense Simulator
 URDF description and Gazebo plugins to simulate Robosense laser scanners. Cloned and adapted from [velodyne_simulator](https://bitbucket.org/DataspeedInc/velodyne_simulator)
 
+This version was authored by Jerome Justin for ROS2 humble version.
+
+NOTE - This version has no GPU capabilities.
```
Comment (tomlogan501) [inline README.md:6]: What is the performance you obtain ?

Comment (jclinton830) [reply README.md:6]: Not great without the GPU. I just wanted something to work for now. I can add the GPU capabilities later. I should have some more time to work on this after two more weeks.

Comment (jclinton830) [reply README.md:6]: I do not have a GPU test at the moment anyway, because I am travelling.

**Pull request merged**